llama_stack/providers/remote/inference/ollama/ollama.py

```diff
@@ -64,27 +65,27 @@
         "llama3.2:3b-instruct-fp16",
         CoreModelId.llama3_2_3b_instruct.value,
     ),
-    build_model_alias(
+    build_model_alias_with_just_provider_model_id(
         "llama3.2:1b",
         CoreModelId.llama3_2_1b_instruct.value,
     ),
-    build_model_alias(
+    build_model_alias_with_just_provider_model_id(
         "llama3.2:3b",
         CoreModelId.llama3_2_3b_instruct.value,
     ),
-    build_model_alias(
+    build_model_alias_with_just_provider_model_id(
         "llama-guard3:8b",
         CoreModelId.llama_guard_3_8b.value,
     ),
-    build_model_alias(
+    build_model_alias_with_just_provider_model_id(
         "llama-guard3:1b",
         CoreModelId.llama_guard_3_1b.value,
     ),
     build_model_alias(
         "x/llama3.2-vision:11b-instruct-fp16",
```

Comment:
oh we should remove the "x/" prefix here. those models are now out released broadly by ollama.

Reply:
done.